CONTRIBUTING.md

```diff
@@ -32,16 +32,14 @@ If you're worried or don’t know where to start, check out our next section exp
 -   Alternatively you can use your machine's LAN IP.
 
 7. Deploy the database with `npm run prisma:migrate`
-8. Start the backend with `npm run start`
-9. Go to the `web/crux-ui` directory: `cd web/crux-ui`
-10. Install dependencies `npm ci`
-11. Copy the _env.example_ file as _.env_ `cp .env.example .env`
-12. Start the frontend with `npm run start`
-13. After you navigated to `localhost:8000` (this is the default Traefik port) you will see a Login screen
-14. Register an account with whatever e-mail address you see fit (doesn't have to be valid one)
-15. Navigate to `localhost:4436` where you will find your mail as all outgoing e-mails will land here
-16. Open your e-mail message and using the link inside you can activate your account
-17. Fruitful contributing! 🎬
+8. Start the backend in development mode with `npm start`
+9. Repeat steps 3-5 in the `web/crux-ui` folder in a different terminal
```

Comment:
Steps 3-5 does not fully apply to `crux-ui`. There's no `DATABASE_URL` neither `CRUX_AGENT_ADDRESS`, only `DNS_DEFAULT_RESULT_ORDER` applies here. So I'm not really sure if step 9. is correct this way.

For step 9, I'd prefer the original text, but replacing `npm run start` with `npm start` and mentioning the fact that you need a different terminal for the backend and frontend is a nice addition though.

I think the UI only needs renaming the .env file and running it, if you could rephrase it that would be cool.

Reply:
Yup , I agree.